style.css

```diff
@@ -331,6 +339,59 @@ section ul .linked-in,
   background-color: #1d1d9e;
 }
 
+/* -- -------------------------------Styling The Popup Window---------------------------------- -- */
+.hide {
+  display: none;
+}
+
+.popupCard {
+  display: flex;
+  flex-direction: column;
+  gap: 14px;
+}
+
+.marginBottom {
+  margin-bottom: 16px;
+}
+
+.paddingBottom {
+  padding-bottom: 12px;
+}
+
+.projectHeader {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 12px;
+}
+
+.closeBtn {
+  font-size: 26px;
+  color: #67798e;
+}
+
+.roleDetail {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 12px;
+}
+
+.skillsBtns {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 12px;
+}
+
+.previewBtns {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  border-top: 1px solid #ebecf0;
+  padding-top: 30px;
+}
+
```

Comment:
- You did a wonderful job styling your popup :100:, but I've found some issues in the style, listed below, kindly remember that you should stick as much as possible to the Figma template :+1: 

- [ ] When the screen size grows please ensure that your popup has a `max-width` and `max-height` defined to avoid it taking more than the required space :+1: 

**Your design**

![Screenshot from 2023-02-02 06-37-17](https://user-images.githubusercontent.com/105117832/216315590-58c0aac3-872a-41d7-8f4b-3d8d4f1818ed.png)

**Figma**

![Screenshot from 2023-02-02 06-35-29](https://user-images.githubusercontent.com/105117832/216315666-523a371a-d775-496b-89f1-7b691ce17bfb.png)

- [ ] I've noticed that there is a lot of empty space below your popup, kindly fix it because it would improve the user experience :+1: 

**Your design**

![Screenshot from 2023-02-02 06-43-58](https://user-images.githubusercontent.com/105117832/216316292-a2b2023f-f167-4b47-bc2b-50059a01730f.png)

![Screenshot from 2023-02-02 06-46-19](https://user-images.githubusercontent.com/105117832/216316733-1f38e33e-8fb5-4e1a-99d5-d97d94cda4fe.png)

- [ ] I have noticed that your image in the popup and also in the work section is changing when you change from the desktop version to the mobile, I think it would be better if they are the same :+1: 

**Before**

![Screenshot from 2023-02-02 06-47-40](https://user-images.githubusercontent.com/105117832/216317180-043b4272-1ddc-4a39-945d-f7379042995d.png)

**After**

![Screenshot from 2023-02-02 06-46-52](https://user-images.githubusercontent.com/105117832/216317297-e96ff5dc-fa8f-45b7-9098-22803af7245e.png)

- [ ] Kindly please be sure to use the same background as the Figma template for the button in the mobile version :+1: 

**Your design**

![Screenshot from 2023-02-02 06-52-55](https://user-images.githubusercontent.com/105117832/216317670-20a3e9f6-b506-4ff3-ba3c-b37cc4e3ebe5.png)

**Figma**

![Screenshot from 2023-02-02 06-53-36](https://user-images.githubusercontent.com/105117832/216317853-87a10778-25b7-4eb6-98bc-ec479c8b9872.png)
 

- [ ] Finally, to match with the Figma your popup design for mobile should have the required background :+1: 

**Your design**

![Screenshot from 2023-02-02 06-46-52](https://user-images.githubusercontent.com/105117832/216318155-34f89c5d-6bfd-43d6-a327-c62cc30a8c3e.png)

**Figma design**

![Screenshot from 2023-02-02 06-53-58](https://user-images.githubusercontent.com/105117832/216318212-3344b04e-eb9a-4a63-8a14-71708438ead0.png)